File: Company_wise_Mock/Microsoft/Day of the Week.js
/**
 * @param {number} day
 * @param {number} month
 * @param {number} year
 * @return {string}
 */
var dayOfTheWeek = function(day, month, year) {
    let d = year.toString() + "-" + month.toString() + "-" + day.toString();
    let date = new Date(d);
    let weekdays = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"];
    
    return weekdays[date.getDay()];
};

File: Algorithms/Day of the Week.py
import datetime

class Solution:
    def dayOfTheWeek(self, day: int, month: int, year: int) -> str:
        x = datetime.datetime(year, month, day)
        weekdays = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"]
        
        return weekdays[int(x.strftime("%w"))]
